app.py

```diff
@@ -0,0 +1,71 @@
+from flask import Flask, jsonify, request
+from flask_sqlalchemy import SQLAlchemy
+
+app = Flask(__name__)
+
+app.config['SQLALCHEMY_DATABASE_URI'] = "postgresql://postgres:farcry1234@localhost/articles"
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+db = SQLAlchemy(app)
+

```

Comment:
Try to modularise your code in modules.
apps.py
views/handler.py
models.py
utils.py